src/app/components/Timestamp/index.jsx

```diff
@@ -13,7 +16,12 @@ const StyledTimestamp = styled.span`
   color: ${C_CLOUD_DARK};
   display: block;
   font-family: ${GEL_FF_REITH_SANS};
-  padding-bottom: ${GEL_SPACING_DBL};
+  &:nth-child(1) {
+    padding-bottom: ${GEL_SPACING_HLF};
```

Comment:
Soooo this will apply `padding-bottom` to the first timestamp in the page, and you then target the second one in the page with a `last-child` to increase its `padding-bottom`.

What happens if we introduce a third one? We'd have to add `&:nth-child(2)`, and so on.

Think it's better if you just set a default here (i.e. remove the `&:nth-child(1) {` and just keep the `padding-bottom: ${...`, then we only override where necessary. 👍

Reply:
Could you suggest the exact changes you mean please?
The reason I added `&:nth-child(1)` is to [match the designs](https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5c596fad65328069be52d487) which have 0.25rem between the two timestamps.
So with `&:nth-child(1)` we have:

![Screen Shot 2019-03-22 at 10 08 05](https://user-images.githubusercontent.com/19362408/54815975-1354cb00-4c8b-11e9-94cc-342134d98405.png)

and without we have:
![Screen Shot 2019-03-22 at 10 07 12](https://user-images.githubusercontent.com/19362408/54815988-1b146f80-4c8b-11e9-884f-b4b34fca21d3.png)

I mean, I can barely tell the difference, but maybe the designers can..

Reply:
```css
  padding-bottom: ${GEL_SPACING_HLF};
  &:last-child {
    padding-bottom: ${GEL_SPACING_DBL};
  }
```

Reply:
Should give the same effect ^

Reply:
got it, thanks!